__tests__/endpoints.test.js

```diff
@@ -52,3 +52,40 @@ describe("GET /api", () => {
       });
   });
 });
+
+// GET /api/articles/:article_id
+describe('GET /api/articles', () => { 
+  test('should return article with a specified id', () => { 
+    return query(app)
+      .get('/api/articles/1')
+      .expect(200)
+      .then((res) => {
+          const article = res.body;
+          expect(typeof article.article_id).toBe('number');
+          expect(typeof article.topic).toBe('string');
+          expect(typeof article.author).toBe('string');
+          expect(typeof article.body).toBe('string');
+          expect(typeof article.created_at).toBe('string');
+          expect(typeof article.votes).toBe('number');
+          expect(typeof article.article_img_url).toBe('string');
```

Comment:
This isn't checking that we are getting the correct article back and could be getting any article.
Start looking into some jest matchers which might be more useful here.
ObjectContaining and ToMatchObject are two good ones.